pkg/registry/servicecatalog/broker/strategy.go

```diff
@@ -48,6 +53,11 @@ var (
 	_ rest.RESTCreateStrategy = brokerRESTStrategies
 	_ rest.RESTUpdateStrategy = brokerRESTStrategies
 	_ rest.RESTDeleteStrategy = brokerRESTStrategies
+
+	brokerStatusUpdateStrategy = brokerStatusRESTStrategy{
+		brokerRESTStrategies,
+	}
+	_ rest.RESTUpdateStrategy = brokerStatusUpdateStrategy
```

Comment:
make sure our status implements the update strategy. I realize it is tautologically true because it is using the upward implementation, but I like to be explicit. I started it out as it's own thing before I copied how all the other ones I could find are implemented.